test/compilable/chkformat.d

```diff
@@ -1,9 +1,12 @@
 // https://issues.dlang.org/show_bug.cgi?id=20643
 // https://issues.dlang.org/show_bug.cgi?id=20644
+// https://issues.dlang.org/show_bug.cgi?id=21177
 /*
+CC: g++
```

Comment:
Why is this necessary ?

Reply:
I thought that would limit the test for the Glibc runtime, but apparently not. Is there an easy way to achieve this ? (Is glibc limited to Linux ?) If not I may use dshell.

Reply:
See `DISABLED` [here](https://github.com/dlang/dmd/tree/master/test#test-configuration).
But you will need to create another test, as we don't want to disable `chkformat.d` test on Windows...

Reply:
No, that's not possible right now.
Guess you can either use dhsell test or try to fake the error messages without glibc using `pragma(msg, ...)`

Reply:
> try to fake the error messages without glibc using `pragma(msg, ...)`

Nice trick !